pihole-google.txt

```diff
@@ -7754,4 +7754,362 @@ www.tofu.jibemobile.com
+maps.google.co.ck
+maps.google.cl
+maps.google.cm
+maps.google.cn
+maps.google.com.co
+maps.google.co.cr
+maps.google.com.cu
+maps.google.cv
+maps.google.com.cy
+maps.google.cz
+maps.google.de
+maps.google.dj
+maps.google.dk
+maps.google.dm
+maps.google.com.do
+maps.google.dz
+maps.google.com.ec
+maps.google.ee
+maps.google.com.et
+maps.google.fi
+maps.google.com.fj
+maps.google.fm
+maps.google.fr
+maps.google.ga
+maps.google.ge
+maps.google.gf
+www.google.gf
+maps.google.gg
+maps.google.com.gh
+maps.google.com.gi
+maps.google.gl
+maps.google.gm
+maps.google.gp
+maps.google.gr
+maps.google.com.gt
+maps.google.gy
+maps.google.com.hk
+maps.google.hn
+maps.google.hr
+maps.google.ht
+maps.google.hu
+maps.google.co.id
+maps.google.ie
+maps.google.co.il
+maps.google.im
+maps.google.co.in
+maps.google.io
+www.google.io
+maps.google.iq
+maps.google.ir
+www.google.ir
+maps.google.is
+maps.google.it
+maps.google.je
+maps.google.com.jm
+maps.google.jo
+maps.google.co.jp
+maps.google.co.ke
+maps.google.kg
+maps.google.com.kh
+maps.google.ki
+maps.google.co.kr
+maps.google.com.kw
+maps.google.kz
+maps.google.la
+maps.google.com.lb
+maps.google.com.lc
+www.google.com.lc
+maps.google.li
+maps.google.lk
+maps.google.com.ls
+www.google.com.ls
+maps.google.lt
+maps.google.lu
+maps.google.lv
+maps.google.com.ly
+maps.google.co.ma
+maps.google.md
+maps.google.me
+maps.google.mg
+maps.google.mk
+maps.google.ml
+maps.google.mn
+maps.google.ms
+maps.google.com.mt
+maps.google.mu
+maps.google.mv
+maps.google.mw
+maps.google.com.mx
+maps.google.com.my
+maps.google.co.mz
+maps.google.com.na
+maps.google.ne
+maps.google.net
+www.google.net
+maps.google.com.nf
+maps.google.com.ng
+maps.google.com.ni
+maps.google.nl
+maps.google.no
+maps.google.com.np
+maps.google.nr
+maps.google.nu
+maps.google.co.nz
+maps.google.com.om
+maps.google.org
+maps.google.com.pa
+maps.google.com.pe
+maps.google.com.pg
+maps.google.com.ph
+maps.google.com.pk
+maps.google.pl
+maps.google.pn
+maps.google.com.pr
+maps.google.ps
+maps.google.pt
+maps.google.com.py
+maps.google.com.qa
+maps.google.ro
+maps.google.rs
+maps.google.ru
+maps.google.rw
+maps.google.com.sa
+maps.google.com.sb
+maps.google.sc
+maps.google.se
+maps.google.com.sg
+maps.google.sh
+maps.google.si
+maps.google.sk
+maps.google.com.sl
+maps.google.sm
+maps.google.sn
+maps.google.so
+maps.google.st
+maps.google.com.sv
+maps.google.td
+maps.google.tg
+maps.google.com.th
+www.google.com.th
+maps.google.com.tj
+maps.google.tk
+maps.google.tl
+maps.google.tm
+maps.google.com.tn
+www.google.com.tn
+maps.google.to
+maps.google.com.tr
+maps.google.tt
+maps.google.com.tw
+maps.google.co.tz
+maps.google.com.ua
+maps.google.co.uk
+maps.google.us
+www.google.us
+maps.google.com.uy
+maps.google.co.uz
+maps.google.com.vc
+maps.google.co.ve
+maps.google.vg
+maps.google.co.vi
+maps.google.com.vn
+maps.google.vu
+maps.google.ws
+maps.google.co.za
+maps.google.co.zm
+maps.google.co.zw
+fcmconnection.googleapis.com
+redirector.googlevideo.com
```

Comment:
```suggestion
yellowstone.jibemobile.com
# More
discord-attachments-uploads-prd.storage.googleapis.com
cs.opensource.google
gnpfesdk-pa.googleapis.com
voilatile-pa.googleapis.com
android-context-data.googleapis.com
appsgrowthpromo-pa.clients1.google.com
appsgrowthpromo-pa.clients2.google.com
appsgrowthpromo-pa.clients3.google.com
appsgrowthpromo-pa.clients4.google.com
appsgrowthpromo-pa.clients5.google.com
appsgrowthpromo-pa.clients6.google.com
doc-0g-48-docstext.googleusercontent.com
maps.google.com.mm
aswpsdkeu.com
drive-thirdparty.googleusercontent.com
gz0.googleusercontent.com
t0.gstatic.com
businessmessages.google
www.tiltbrush.com
0.android.pool.ntp.org
1.android.pool.ntp.org
2.android.pool.ntp.org
3.android.pool.ntp.org
4.android.pool.ntp.org
5.android.pool.ntp.org
6.android.pool.ntp.org
7.android.pool.ntp.org
8.android.pool.ntp.org
9.android.pool.ntp.org
android.apis.google.com
android-developers.googleblog.com
android.pool.ntp.org
androidenterprisepartners.withgoogle.com
3cbc684bb9f2dcdd3a3794de9de52f63.safeframe.googlesyndication.com
addons-pa.clients1.google.com
addons-pa.clients2.google.com
addons-pa.clients3.google.com
addons-pa.clients4.google.com
addons-pa.clients5.google.com
addons-pa.clients6.google.com
appsitemsuggest-pa.clients1.google.com
appsitemsuggest-pa.clients2.google.com
appsitemsuggest-pa.clients3.google.com
appsitemsuggest-pa.clients4.google.com
appsitemsuggest-pa.clients5.google.com
appsitemsuggest-pa.clients6.google.com
appswaldo.clients1.google.com
appswaldo.clients2.google.com
appswaldo.clients3.google.com
appswaldo.clients4.google.com
appswaldo.clients5.google.com
appswaldo.clients6.google.com
aswpsdkeu.com see whois record
blobcomments-pa.clients1.google.com
blobcomments-pa.clients2.google.com
blobcomments-pa.clients3.google.com
blobcomments-pa.clients4.google.com
blobcomments-pa.clients5.google.com
carrers.google.com
ci1.googleusercontent.com
ci2.googleusercontent.com
content-firebaseappcheck.googleapis.com
datacompute.google.com
encrypted-vtbn0.gstatic.com
espresso-pa.clients1.google.com
espresso-pa.clients2.google.com
espresso-pa.clients3.google.com
espresso-pa.clients4.google.com
espresso-pa.clients5.google.com
espresso-pa.clients6.google.com
fb50436e04f648465c9ba6a8892caa86.safeframe.googlesyndication.com
feedback-pa.clients1.google.com
feedback-pa.clients2.google.com
feedback-pa.clients3.google.com
feedback-pa.clients4.google.com
feedback-pa.clients5.google.com
feedback-pa.clients6.google.com
instantmessaging-pa-eu.googleapis.com
jnn-pa.googleapis.com
mail-attachment.googleusercontent.com
newsstand.googleusercontent.com
opensource.googleblog.com
opensourcelive.withgoogle.com
people-pa.clients1.google.com
people-pa.clients2.google.com
people-pa.clients3.google.com
people-pa.clients4.google.com
people-pa.clients5.google.com
people-pa.clients6.google.com
peoplestack-pa.clients1.google.com
peoplestack-pa.clients2.google.com
peoplestack-pa.clients3.google.com
peoplestack-pa.clients4.google.com
peoplestack-pa.clients5.google.com
peoplestack-pa.clients6.google.com
peoplestackwebexperiments-pa.clients1.google.com
peoplestackwebexperiments-pa.clients2.google.com
peoplestackwebexperiments-pa.clients3.google.com
peoplestackwebexperiments-pa.clients4.google.com
peoplestackwebexperiments-pa.clients5.google.com
pop.gmail.com
scone-pa.clients1.google.com
scone-pa.clients2.google.com
scone-pa.clients3.google.com
scone-pa.clients4.google.com
scone-pa.clients5.google.com
scone-pa.clients6.google.com
signaler-pa.clients1.google.com
signaler-pa.clients2.google.com
signaler-pa.clients3.google.com
signaler-pa.clients4.google.com
signaler-pa.clients5.google.com
smtp.gmail.com
streetviewpixels-ga.googleapis.com
suggestqueries-clients1.youtube.com
suggestqueries-clients2.youtube.com
suggestqueries-clients3.youtube.com
suggestqueries-clients4.youtube.com
suggestqueries-clients5.youtube.com
suggestqueries-clients6.youtube.com
s-usc1a-nss-2023.firebase.io
tables.area120.google.com
taskassist-pa.clients1.google.com
taskassist-pa.clients2.google.com
taskassist-pa.clients3.google.com
taskassist-pa.clients4.google.com
taskassist-pa.clients5.google.com
taskassist-pa.clients6.google.com
webdesigner.withgoogle.com
www.googlee.com
www.googlemaps.com
d3e54v103j8qbb.cloudfront.net
cdn.fastclick.net
media.fastclick.net
yt4.ggpht.com
biz.waze.com
waze-app.com
wazeopedia.waze.com
waze.uservoice.com
worldtiles1.waze.com
maps.google.ac
maps.google.ad
maps.google.ae
maps.google.com.af
maps.google.com.ag
maps.google.com.ai
maps.google.al
maps.google.am
maps.google.co.ao
maps.google.com.ar
maps.google.as
maps.google.at
maps.google.com.au
maps.google.az
maps.google.ba
maps.google.com.bd
maps.google.be
maps.google.bf
maps.google.bg
maps.google.com.bh
maps.google.bj
maps.google.com.bn
maps.google.com.bo
maps.google.bs
maps.google.bt
maps.google.co.bw
maps.google.by
maps.google.com.bz
maps.google.ca
maps.google.cat
maps.google.cc
www.google.cc
maps.google.cd
maps.google.cf
maps.google.cg
maps.google.ch
maps.google.ci
maps.google.co.ck
maps.google.cl
maps.google.cm
maps.google.cn
maps.google.com.co
maps.google.co.cr
maps.google.com.cu
maps.google.cv
maps.google.com.cy
maps.google.cz
maps.google.de
maps.google.dj
maps.google.dk
maps.google.dm
maps.google.com.do
maps.google.dz
maps.google.com.ec
maps.google.ee
maps.google.com.et
maps.google.fi
maps.google.com.fj
maps.google.fm
maps.google.fr
maps.google.ga
maps.google.ge
maps.google.gf
www.google.gf
maps.google.gg
maps.google.com.gh
maps.google.com.gi
maps.google.gl
maps.google.gm
maps.google.gp
maps.google.gr
maps.google.com.gt
maps.google.gy
maps.google.com.hk
maps.google.hn
maps.google.hr
maps.google.ht
maps.google.hu
maps.google.co.id
maps.google.ie
maps.google.co.il
maps.google.im
maps.google.co.in
maps.google.io
www.google.io
maps.google.iq
maps.google.ir
www.google.ir
maps.google.is
maps.google.it
maps.google.je
maps.google.com.jm
maps.google.jo
maps.google.co.jp
maps.google.co.ke
maps.google.kg
maps.google.com.kh
maps.google.ki
maps.google.co.kr
maps.google.com.kw
maps.google.kz
maps.google.la
maps.google.com.lb
maps.google.com.lc
www.google.com.lc
maps.google.li
maps.google.lk
maps.google.com.ls
www.google.com.ls
maps.google.lt
maps.google.lu
maps.google.lv
maps.google.com.ly
maps.google.co.ma
maps.google.md
maps.google.me
maps.google.mg
maps.google.mk
maps.google.ml
maps.google.mn
maps.google.ms
maps.google.com.mt
maps.google.mu
maps.google.mv
maps.google.mw
maps.google.com.mx
maps.google.com.my
maps.google.co.mz
maps.google.com.na
maps.google.ne
maps.google.net
www.google.net
maps.google.com.nf
maps.google.com.ng
maps.google.com.ni
maps.google.nl
maps.google.no
maps.google.com.np
maps.google.nr
maps.google.nu
maps.google.co.nz
maps.google.com.om
maps.google.org
maps.google.com.pa
maps.google.com.pe
maps.google.com.pg
maps.google.com.ph
maps.google.com.pk
maps.google.pl
maps.google.pn
maps.google.com.pr
maps.google.ps
maps.google.pt
maps.google.com.py
maps.google.com.qa
maps.google.ro
maps.google.rs
maps.google.ru
maps.google.rw
maps.google.com.sa
maps.google.com.sb
maps.google.sc
maps.google.se
maps.google.com.sg
maps.google.sh
maps.google.si
maps.google.sk
maps.google.com.sl
maps.google.sm
maps.google.sn
maps.google.so
maps.google.st
maps.google.com.sv
maps.google.td
maps.google.tg
maps.google.com.th
www.google.com.th
maps.google.com.tj
maps.google.tk
maps.google.tl
maps.google.tm
maps.google.com.tn
www.google.com.tn
maps.google.to
maps.google.com.tr
maps.google.tt
maps.google.com.tw
maps.google.co.tz
maps.google.com.ua
maps.google.co.uk
maps.google.us
www.google.us
maps.google.com.uy
maps.google.co.uz
maps.google.com.vc
maps.google.co.ve
maps.google.vg
maps.google.co.vi
maps.google.com.vn
maps.google.vu
maps.google.ws
maps.google.co.za
maps.google.co.zm
maps.google.co.zw
fcmconnection.googleapis.com
```

The failing test is for inculding the [regex](https://github.com/nickspaargaren/no-google#warning-important-regex-filters-not-included-in-blocklist) domains, can you remove these?